tests/test_cli.py

```
@@ -0,0 +1,209 @@
+    result = runner.invoke(
+        app, ["update-strategist", "--new-strategist-address", "0xNewStrategist"]
+    )
+
+    assert result.exit_code == 0
+    assert "Strategist address updated successfully" in result.stdout
+
+
+@patch("orion_finance_sdk_py.cli.OrionTransparentVault")
+@patch("orion_finance_sdk_py.cli.ensure_env_file")
+@patch.dict("os.environ", {"ORION_VAULT_ADDRESS": "0xVault"})
+def test_update_fee_model(mock_ensure, MockVault):
```

Ruff (0.14.10)

Unused function argument: `mock_ensure`

(ARG001)